### ISSUE TITLE
Question about mc_dropout_evaluate

### ISSUE THREAD

**Issue by jianzhengming** [microsoft/UST#2]
In this function, you set the number  of masked models for uncertainty estimation as 30. 

However, I cannot find how to change the weight in the loop:

    for i in range(T):

        y_pred = []
        with strategy.scope():
            def eval_step(inputs):
                return model(inputs, training=training).numpy()#[:,0]

            def distributed_eval_step(dataset_inputs):
                return strategy.experimental_run_v2(eval_step, args=(dataset_inputs,))

            for batch in dist_data:
                pred = distributed_eval_step(batch)
                for gpu in range(gpus):
                    y_pred.extend(pred.values[gpu])

        #converting logits to probabilities
        y_T[i] = tf.nn.softmax(np.array(y_pred))

Did I misunderstand the  T stochastic forward passes?

**Comment by subhomj** [comment]
Since dropout is enabled at inference with `training=training`, each of the passes `i \in T` activates (1-p)% of  the network, with p being the dropout probability. The corresponding stochastic predictions are accumulated in y_T and correspondingly used as soft labels for training.
